X16 Reference - 02 - Editor.md

```diff
@@ -434,8 +434,27 @@ keyhandler:
     jsr $ffd2
 
 exit:
-    plx		;Restore input
+    plx     ;Restore input
     pla
     plp
     rts
 ```
+
+### Function Key Shortcuts
+
+The following Function key macros are pre-defined for your convenience. These shortcuts only work in immediate mode. When a program is running, the F-keys generate the corresponding PETSCII character code.
```

Comment:
I guess we should probably be consistent in terminology.  Other references have called it "direct mode" rather than immediate mode.

Reply:
Immediate Mode is the canonical name for typing directly into the editor. 

page 32: 
https://www.commodore.ca/manuals/c64_users_guide/c64-users_guide-03-beginning_basic_programming.pdf

Direct Mode is used the GW-BASIC documentation, so it really comes down to our preferences, I guess. I have always used the term "immediate mode", because this is what Commodore used.

Reply:
I guess it's inconsistent

![image](https://user-images.githubusercontent.com/395186/226070394-f9c6514d-c1f1-477a-9bfd-c53dbf4e3f8a.png)

https://www.c64-wiki.com/wiki/Direct_Mode